src/Behat/Behat/Output/Node/Printer/JUnit/JUnitFeaturePrinter.php

```diff
@@ -68,6 +68,7 @@ public function printFooter(Formatter $formatter, TestResult $result)
             'failures' => $stats[TestResult::FAILED],
             'errors' => $stats[TestResult::PENDING] + $stats[TestResult::UNDEFINED],
             'time' => $this->durationListener ? $this->durationListener->getFeatureDuration($this->currentFeature) : '',
+            'file' => $this->currentFeature->getFile(),
```

Comment:
I think for consistency this would need to be a relative path the same as the `file` attribute on the `<testcase` node.

Currently that is calculated internal to the JUnitScenarioPrinter https://github.com/Behat/Behat/blob/610fa32937bf52bd79d812103464600ef4649e36/src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php#L86-L91 but it would probably be better to extract that rather than duplicating it...

Note also that it appears Gherkin does not guarantee the file is defined - `->getFile()` can be `null|string`. I have to say I'm not sure why/when it could be missing... The scenario printer therefore only adds a `file` attribute if there is a file to report.